craft_store/base_client.py

```diff
@@ -251,3 +251,10 @@ def progress_printer(monitor: requests_toolbelt.MultipartEncoderMonitor):
         logger.debug("Uploading bytes for %r ended, id %r", str(filepath), upload_id)
 
         return upload_id
+
+    def get_list_releases(self, *, name: str) -> models.MarshableModel:
+        """Query the list_releases endpoint and return the result."""
+        endpoint = f"/v1/{self._endpoints.namespace}/{name}/releases"
+        response = self.request("GET", self._base_url + endpoint).json()
+
+        return self._endpoints.get_list_releases.unmarshal(response)
```

Comment:
I initially thought `Endpoints.get_list_releases` was a function, but it is the expected response model (confusingly, it *is* a `BaseClient` method, too). I'd suggest maybe drop the `get` and add `model`? E.g. `list_releases_model`, `release_model`, `list_releases_item_model`, etc.

Reply:
this is a good observation!